src/langchain_google_alloydb_pg/vectorstore.py

```diff
@@ -499,6 +546,57 @@ async def afrom_documents(  # type: ignore[override]
         await vs.aadd_texts(texts, metadatas=metadatas, ids=ids, **kwargs)
         return vs
 
+    @classmethod
+    async def afrom_images(  # type: ignore[override]
```

Comment:
I don't love the "from_" methods, so I may have us hold off here until this is enforced by the interface

Reply:
removed